### DISCUSSION TITLE
SSH: where are my other apps?

### DISCUSSION THREAD

**Discussion by c-dilworth** [caprover/caprover#1319] [Q&A]
Hi, I installed the 1-click SSH app hoping it would give me SSH access to my other installed 1-click apps, and I can login fine using the SSH app short-code and the short-codes for my other apps, but each time I see the exact same Linux file system (I left a marker.txt file there), and I cannot find my other apps. Please, can anyone tell me how I can access them? Examples of my SSH logins are:
util-ssh.server.mydomain.com
mysql55-db.server.mydomain.com
wp-wordpress.server.mydomain.com
*edited to add examples

**Comment by githubsaturn** [comment]
SSH one click app is just a container. You cannot view other apps from the isolated container. 

What you want is this:
https://caprover.com/docs/troubleshooting.html#how-to-run-shell-inside-my-application-inside-container

**Reply by c-dilworth** [reply]
Thanks for that. It does work, but I wanted to use my SSH client with gui. BTW the docs at https://caprover.com/docs/database-connection.html#external-connections misled me. They say "IMPORTANT: Note that you are not able to SSH Tunnel to your database from the regular SSH on server. You must create an SSH container. SSH on the host is not able to talk to the container.". And they even give a picture showing how the SSH container can talk to a database container.

**Reply by githubsaturn** [reply]
Do you want to shell into your container or connect to it from your laptop? These are different things. What's you intention? What do you want to do?

**Reply by c-dilworth** [reply]
Connect from my laptop using my SSH client with gui.

**Reply by githubsaturn** [reply]
Then you have two solutions:
1- If you have fairly familiar with SSH port forwarding and dockerized containers, you can try:
```
ssh -L 8181:srv-captain--mysql:3306 root@<ip of your CapRover Server> -p 4646
```
Then on your laptop, you can connect to localhost:8181
Under the hood localhost:8181 => Server:4646 => database:3306

2- If you are not too familiar with how this works, you should completely ignore the SSH path, just simply use port forwarding approach outlined here:
https://caprover.com/docs/database-connection.html#external-connections

**Reply by c-dilworth** [reply]
Thanks for that. I'll figure it out from there. BTW, despite what the docs say, I can simply shell into my host and go to /var/lib/docker/volumes/captain--myappname and i have full access to the container's file system through the cli and sftp, but I prefer a gui.

**Reply by githubsaturn** [reply]
What you're seeing is accessing the file system... not the app... Again these are very different things!

If you only need access to the files, you can just use any SSH gui client and just SSH to the machine.
